RMv4_Recipely/RMv4_Recipely/Network Service Layer/HitDTO.swift

```diff
@@ -0,0 +1,10 @@
+// HitDTO.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// DTO хита
```

Comment:
```suggestion
/// DTO хита
```
что за хит? песня какая-то?)